### DISCUSSION TITLE
Will support be added for the New Places API when it becomes available?

### DISCUSSION THREAD

**Discussion by tyeetale** [wellyshen/use-places-autocomplete#1138] [Q&A] [answered]
Hi!

So I noticed the recent Google Place API updates, featuring more fields and improved performance. However, the current support is limited to cURL, Android, and iOS, with no JavaScript API yet.

Any plans for use-places-autocomplete to support the new Place API in the future? Given Google's quick deprecation tendencies, it'd be great to stay ahead.

**Comment by wellyshen** [comment, marked as answer]
I didn't see the related JS library at this stage. If it comes out, we will support it 😎

**Reply by FilipLjubic** [reply]
are there still plans of supporting it?

**Comment by sreekwork** [comment]
Hi, Any update on supporting the New places API?
